src/main/scala/dsptools/counters/Counters.scala

```diff
@@ -66,21 +66,21 @@ class CountIO (countParams: CountParams) extends Bundle {
   // Count up/down control signal
   val upDown = if (countParams.countType == UpDown) Some(Input(Bool())) else None
   // Counters usually increment by 1
-  val inc = if (countParams.incMax != 1) Some(Input(UInt(countParams.incMax))) else None
```

Comment:
This seems like a literal rather than a width?
Though the original code may have been just wrong, since you shouldn't have a literal inside an input...

Similarly for the next few instances.